geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt

```diff
@@ -28,6 +28,20 @@ Information
    ``service: geoportal`` by ``service: geoportal-advance``, and in service ``alembic`` you should replace
    ``service: alembic`` by ``service: alembic-advance``.
 
+4. Some little changes in the application build:
+   - We no more use custom build commands to build the docker images, we simply use the docker-compose
+     build command, with tow additional environment variable:
+     * ``SIMPLE``: ``TRUE`` or ``FALSE`` if we compile a simple application or not.
+     * ``GIT_HASH``: the git hash of the current version.
+     Then all the build specification will be moved to the ``docker-compose`` files.
+   - The ``--geoportal`` and ``--config``arguments will be replaced by ``--service=geoportal`` and
+     ``--service=config`` respectively.
+   - The image used by the ``docker-compose`` will be pulled from the Docker hub.
+   - New argument ``--fast-reload`` that will recreate all containers and restart them except the redis once
```

Comment:
"the redis once"
we probably mean "the redis containers"? or "the redis ones"
but "the redis containers" is clearer